--- jakarta-tomcat-connectors-1.2.14.1-src/jk/native/scripts/build/rules.mk~	2005-06-15 14:10:39.000000000 +0300
+++ jakarta-tomcat-connectors-1.2.14.1-src/jk/native/scripts/build/rules.mk	2005-12-26 21:47:33.000000000 +0200
@@ -4,7 +4,7 @@
 # Compile commands
 #VPATH=.:../common
 COMPILE      = $(CC) $(CFLAGS)
-LT_COMPILE   = $(LIBTOOL) --mode=compile $(COMPILE) -c $< -o $@
+LT_COMPILE   = $(LIBTOOL) --mode=compile --tag=CC $(COMPILE) -c $< -o $@
 
 # Implicit rules for creating outputs from input files
 
--- jakarta-tomcat-connectors-1.2.14.1-src/jk/native/apache-2.0/Makefile.in~	2005-12-27 01:53:31.000000000 +0200
+++ jakarta-tomcat-connectors-1.2.14.1-src/jk/native/apache-2.0/Makefile.in	2005-12-27 01:54:41.000000000 +0200
@@ -63,7 +63,7 @@
 # APXS will compile every file, this is derived from apxs
 
 mod_jk.la: mod_jk.lo $(APACHE_OBJECTS)
-	$(LIBTOOL) --mode=link ${COMPILE} `${APXS} -q LDFLAGS` -o $@ -module -rpath ${libexecdir} -avoid-version mod_jk.lo $(APACHE_OBJECTS)
+	$(LIBTOOL) --mode=link --tag=CC ${COMPILE} `${APXS} -q LDFLAGS` -o $@ -module -rpath ${libexecdir} -avoid-version mod_jk.lo $(APACHE_OBJECTS)
 
 mod_jk.so: mod_jk.la
 	$(LIBTOOL) --mode=install cp $< `pwd`/$@
--- jakarta-tomcat-connectors-1.2.14.1-src/jk/native/apache-2.0/Makefile.in~	2005-03-29 17:35:48.000000000 +0300
+++ jakarta-tomcat-connectors-1.2.14.1-src/jk/native/apache-2.0/Makefile.in	2005-12-27 01:53:31.000000000 +0200
@@ -42,7 +42,7 @@
 	( cd ..; ./config.status )
 
 lib_jk.la: mod_jk.lo ${APACHE_OBJECTS}
-	$(LIBTOOL) --mode=link $(CC) -o lib_jk.la -static -rpath ${libexecdir}/jk mod_jk.lo $(APACHE_OBJECTS)
+	$(LIBTOOL) --mode=link --tag=CC $(CC) -o lib_jk.la -static -rpath ${libexecdir}/jk mod_jk.lo $(APACHE_OBJECTS)
 
 install_static:
 	@echo ""
